USF-2278: initial draft of changes for removing references to gdrive and sharepoint (#277)
commerce-docs/microsite-commerce-storefront: src/content/docs/merchants/get-started/multistore.mdx

### Content files

> The content files provide the structure of the pages served to your shoppers. They are located in the store view directories and contain the content for each store view. These files are documents on https://da.live, SharePoint `.docx`, or Google document files used to generate the store view pages. These files should be localized or translated into the desired language.

Reviewer: ```suggestion
The content files provide the structure of the pages served to your shoppers. They are located in the store view directories and contain the content for each store view. These files are documents hosted on `https://da.live` (Document Author environment), SharePoint `.docx`, or Google that are used to generate the store view pages. These files should be localized for the desired region and language.
```

### Placeholders

> The `placeholders` sheet stores reusable variables for text and UI components. This file is used to define the text that is displayed on your storefront. The `placeholders.json` file is generated from this file and is used to replace the variables in your content files.

Reviewer: ```suggestion
The placeholders sheet defines reusable variables for text and UI components displayed on your storefront. It is used to generate the `placeholders.json` file, which replaces these variables in your content files.
```
